parquet-format-structures/src/test/java/org/apache/parquet/format/TestUtil.java

```diff
@@ -77,6 +81,21 @@ public void testReadFileMetadata() throws Exception {
     assertEquals(md, md6);
   }
 
+  @Test
+  public void testInvalidPageHeader() throws IOException {
+    PageHeader ph = new PageHeader(PageType.DATA_PAGE, 100, -50);
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    Util.writePageHeader(ph, out);
+
+    try {
+      Util.readPageHeader(in(out));
+      fail("Expected exception but did not thrown");
+    } catch (InvalidParquetMetadataException e) {
+      assertTrue("Exception message does not contain the expected parts",
+          e.getMessage().contains("pageHeader.compressed_page_size"));
```

Comment:
Isn't there an assertion helper so you don't need to catch the exception? Something like assertThrows in the codebase?

Reply:
Yes, there is something already implemented but in another module and I cannot use it here.